examples/enterprise-azure-deployment/enterprise-workspace-deploy/main.tf

```diff
@@ -0,0 +1,91 @@
+provider "azurerm" {
+  client_id         = var.client_id
+  client_secret     = var.client_secret
+  tenant_id         = var.tenant_id
+  subscription_id   = var.subscription_id
+  features {}
+}
+
+resource "azurerm_databricks_workspace" "enterprise-demo-1" {
+  location                      = var.region
+  name                          = var.dbws_name
+  resource_group_name           = var.dbws_rg_name
+  managed_resource_group_name   = var.managed_rg_name
+  sku                           = "premium"
+  custom_parameters {
+    no_public_ip = "true"
+    public_subnet_name = var.subnet_public
+    private_subnet_name = var.subnet_private
+    virtual_network_id = var.vnet_id
+  }
+  tags = {
+    Owner = "jason.robey@databricks.com"
+  }
```

Comment:
you may want to move this out of the example

Reply:
doh!